mtgjson5/providers/tcgplayer.py

```diff
@@ -256,8 +269,178 @@ def generate_today_price_dict(
 
         return dict(combined_listings)
 
+    def determine_mtgjson_sealed_product_category(
+        self, product_name: str
+    ) -> MtgjsonSealedProductCategory:
+        """
+        Best-effort to parse the product name and determine the sealed product category
+        :param product_name Name of the product from TCG
+        :return: category
+        """
+
+        # The order of the checks is important
+        for tag in [
```

Comment:
Could simplify this with `if any(tag for tag in list): return CASE`

Reply:
ok